dev/tasks/tasks.yml

```diff
@@ -1522,6 +1525,9 @@ tasks:
     ci: github
     template: docker-tests/github.cuda.yml
     params:
+      env:
+        UBUNTU: 22.04
+        CUDA: 11.8.0
```

Comment:
Can we keep the same UBUNTU and CUDA version for both jobs?

Reply:
I've reverted back to defaults. This was just some testing we were doing last week when we tried to update the runners and it was failing